modules/openapi-generator/src/main/resources/python/model_test.mustache

```diff
@@ -22,11 +24,20 @@ class Test{{classname}}(unittest.TestCase):
     def tearDown(self):
         pass
 
-    def test{{classname}}(self):
+    def make_instance(self, include_optional):
         """Test {{classname}}"""
-        # FIXME: construct object with mandatory attributes with example values
         # model = {{packageName}}.models.{{classFilename}}.{{classname}}()  # noqa: E501
-        pass
+        if (include_optional) :
+            return {{classname}}({{#vars}}
+                {{name}} = {{#example}}{{{.}}}{{/example}}{{^example}}None{{/example}}{{#hasMore}}, {{/hasMore}}{{/vars}})  # noqa: E501
+        else:
+            return {{classname}}({{#vars}}{{#required}}
```

Comment:
Same comment as above.
Can we change this to:
```
            return {{classname}}(
{{#vars}}
{{#required}}
                {{name}} = {{#example}}{{{.}}}{{/example}}{{^example}}None{{/example}},
{{/required}}
{{/vars}})
            )
```
That way it is clearer that we are looping over vars
Can we get rid of the # noqa: E501 or do we need it?
Please also remove the hasMore tag like I did because we are referring to specific vars, and if the user turns off sortparamsbyrequired then hasMore will not tell us when to turn off commas, best to leave them all on. This doesn't apply to the above if statement because there we are looping over and using all of vars.

Reply:
Done.